server/src/main/java/org/elasticsearch/index/snapshots/IndexShardSnapshotStatus.java

```diff
@@ -461,6 +482,8 @@ public String toString() {
             + processedSize
             + ", failure='"
             + failure
+            + "', debugStatusString='"
```

Comment:
Not sure about exposing this to users as `debugStatusString`, which I believe might happen since this is logged at `INFO` sometimes. Seems a little overly technical. How about `status` or `statusDescription`?